src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -103,7 +111,19 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
     `;
   }
 
-  private renderStyle() {
+  protected updated(_changedProperties: PropertyValues): void {
+    if (!this._config || !this.hass) {
+      return;
+    }
+    if (
+      !_changedProperties.get("hass") ||
+      (_changedProperties.get("hass") as any).themes !== this.hass.themes
```

Comment:
`any`.

Easier to extract `hass` before this `if`, so you can just do something like `(!oldHass || oldHass.themes !== this.hass.themes)`